pyproject.toml

```diff
@@ -38,3 +38,7 @@ select = ["E", "F", "UP", "I", "B"]
 
 [tool.ruff.lint.isort]
 force-single-line = true
+
+[tool.pytest.ini_options]
```

Comment:
I needed to add this to get pytest to work with uv.

```
uv venv
uv pip compile pyproject.toml -o requirements.txt
uv pip install -r requirements.txt
pytest
```

Let me know if there's a preferred way of running the tests, I can remove this change.

Reply:
Thanks. Haven't played with uv yet, but will do soon. For now, all tests seem to pass with this new config. I will merge.